config/core/base.pan

```diff
@@ -16,16 +16,16 @@ variable SITE_ADDITIONAL_PACKAGES ?= undef;
 variable OS_VERSION_PARAMS ?= nlist(
     "distribution", "el",
     "family",       "el",
-    "major",        "el8",
-    "majorversion", "8",
+    "major",        "el9",
+    "majorversion", "9",
     "minor",        "x",
     "flavour",      "x",
-    "version",      "el8x",
-    "arch",         "x86_64"
+    "version",      "el9x",
+    "arch",         "x86_64",
 );
 
-variable RPM_BASE_FLAVOUR = '8';
-variable RPM_BASE_FLAVOUR_VERSIONID = 8;
+variable RPM_BASE_FLAVOUR = '9';
+variable RPM_BASE_FLAVOUR_VERSIONID = 9;
 variable RPM_BASE_FLAVOUR_NAME = format('el%s',RPM_BASE_FLAVOUR_VERSIONID);
```

Comment:
Should these all be `final`?

Reply:
I think almost all variables in this template should be `final`. There is an historical lack of `final` statements in the template library. Let's start by fixing this one!